portal-api/src/scheduler/jobs/send-report-due-emails-job/index.js

```diff
@@ -0,0 +1,113 @@
+const { format, isSameDay } = require('date-fns');
```

Comment:
All LGTM, only comments will probably be on how it's commonised with the late report job so look forward to that excitement!